styleguide/components/61-layouts/overview-layout/_overview-layout.scss

```diff
@@ -22,4 +22,8 @@
     margin-bottom: 2rem;
     font-size: .9rem;
   }
+
+  > .gent-info {
```

Comment:
Is .full-width available as class too?

Reply:
Not yet; I suppose we could make it available within some layouts, but I'm a bit sceptical about making it available as a utility class, since this opens the door to using it everywhere... 

wdyt?

Reply:
That should work.

The issue is that .detail-layout > 'some random element' doens't work in angular once you start using child components.
But we **can** place classes on the template tags.

Reply:
Your prayers have been answered 🙌